v5/core/container_authenticator.go

```diff
@@ -404,7 +404,7 @@ func (authenticator *ContainerAuthenticator) RequestToken() (*IamTokenServerResp
 		// If the user told us to disable SSL verification, then do it now.
 		if authenticator.DisableSSLVerification {
 			transport := &http.Transport{
-				/* #nosec G402 */
+				// #nosec G402
```

Comment:
For some reason, the latest version of gosec (2.9.4, which was released yesterday and apparently is used by Brent's security scanning) prefers `// #nosec` over `/* #nosec */` for inline code annotations.